define/define.py

```diff
@@ -0,0 +1,36 @@
+import urllib2
+import json
+import random
+
+def get_rhyme(word):
+    request = urllib2.urlopen('https://api.datamuse.com/words?rel_rhy=' + word)
```

Comment:
Since this is now the second thing using Datamuse, I'd like to move it to its own module so it's easier to track what parts of it we're using.